nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/ecm/admin/operation/HotReloadStudioSnapshot.java

```diff
@@ -81,6 +80,10 @@ public void run() throws Exception {
             throw new NuxeoException("Must be Administrator to use this function");
         }
 
+        if (!Framework.isDevModeSet()) {
+            throw new NuxeoException("You must enable Dev mode to Hot reload your Studio Snapshot package.");
+        }
+
```

Comment:
This (and the change of the default value of `validate`) seem to be unrelated to the split. Shouldn't they be in a separate ticket?